ttnn/cpp/ttnn/operations/pool/maxpool/max_pool2d_pybind.hpp

```diff
@@ -11,5 +11,6 @@ namespace py = pybind11;
 namespace ttnn::operations::pool {
 
 void bind_max_pool2d_operation(py::module& module);
+void py_module(py::module& module);
```

Comment:
why do you need this function?

Reply:
it calls the py bind function

Reply:
previously it was called in the old maxpool pybind, so just moving it here